apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -49,6 +49,12 @@ export class SendMessageInApp extends SendMessageType {
       );
     }
 
+    if (inAppChannel.template.cta?.actions) {
+      inAppChannel.template.cta?.actions.map((action) => {
+        if (action.content.text) return contentService.replaceVariables(action.content.text, messageVariables);
+      });
```

Comment:
If you are doing a map, try to return a variable rather a modified array, if you do won't to have a side effect (which I don't really recommend. You should just do a forEach or for loop in this case.

Would suggest to do:
```suggestion
      inAppChannel.template.cta?.actions = inAppChannel.template.cta?.actions.map((action) => {
        return contentService.replaceVariables(action.content.text || '', messageVariables);
      });
```

Reply:
Done